x/ibc-rate-limit/types/errors.go

```diff
@@ -5,8 +5,7 @@ import (
 )
 
 var (
-	RateLimitExceededMsg = "rate limit exceeded"
-	ErrRateLimitExceeded = sdkerrors.Register(ModuleName, 2, RateLimitExceededMsg)
+	ErrRateLimitExceeded = sdkerrors.Register(ModuleName, 2, "rate limit exceeded")
 	ErrBadMessage        = sdkerrors.Register(ModuleName, 3, "bad message")
 	ErrContractError     = sdkerrors.Register(ModuleName, 4, "contract error")
```

Comment:
Lets not register any of these codes imo, and just do normal errors. I think we should move away from code spaced errors for now

Reply:
You mean just use the strings? Eventually we have to convert it to a `NewError` right? which requires the code